operations/reservations.md

```diff
@@ -220,8 +228,7 @@ Returns all reservations specified by any identifier, customer or other filter.
 | `AvailabilityBlockId` | string | optional | Unique identifier of the [Availability block](services.md#availability-block) the reservation is assigned to. |
 | `RateId` | string | required | Identifier of the reservation [Rate](services.md#rate). |
 | `VoucherId` | string | optional | Unique identifier of the [Voucher](services.md#voucher) that has been used to create reservation. |
-| `AdultCount` | number | required | Count of adults the reservation was booked for. |
-| `ChildCount` | number | required | Count of children the reservation was booked for. |
+| `PersonCounts` | array of [Age category person count](#age-category-person-count) | required | Number of people per age category the reservation was booked for. At least one category with valid count must be provided. |
```

Comment:
TBH, I don't like the name when I see it in the documentation. I would rename it here, in the documentation to something like "AgeCategoryAssignment"? @MikeAdamsMews what do you think?

Reply:
What do you mean? Renaming the `PersonCounts`, so we need to rename it also in the Dto,  or you mean renaming the `Age category person count`

Reply:
With  `Age category person count` I aggree, it's not the best name. I'll rename that. Just doublechecking if you mean that.

Reply:
I think 'PersonCounts' is ok, it's clear what this means, but yes 'Age category person count' is a bit of a mouthful!